client/lcd/root.go

```diff
@@ -96,3 +103,100 @@ func createHandler(cdc *wire.Codec) http.Handler {
 
 	return r
 }
+
+// ServeSwaggerCommand will generate a long-running rest server
+// that exposes functionality similar to the ServeCommand, but it provide swagger-ui
+// Which is much friendly for further development
+func ServeSwaggerCommand(cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rest-server-swagger",
```

Comment:
Can we use a different name here? This is going to sit in the `gaiacli` command, so maybe something like `gaiacli lite serve`?